src/decays/H_SM_decays/decay_H_to_uquq.cpp

```diff
@@ -0,0 +1,164 @@
+// template specialization for the H -> Fu Fu case
```

Comment:
I'm a bit unhappy about these longish functions. They appear hard to read for me. Can they be split up into smaller pieces? Maybe the loop corrections can be extracted into smaller functions?

(This applies also to the other template specializations that are this long.)